### ISSUE TITLE
Crafting storage breaks after being placed on network with security terminal

### ISSUE THREAD

**Issue by MilkMC** [Tfarcenim/AE2-Extras#9]
When placing the 16M crafting storage on my ME network, it automatically breaks. I haven't tried any of the other crafting storages added by this mod, but I would assume the same issue applies to all of them. If I place the block first and then attach a cable to the network afterwards it seems to work fine. Here is a gif of the issue: [https://gyazo.com/1affc8e0f33b6d0050ee7b27b0f66a56](https://gyazo.com/1affc8e0f33b6d0050ee7b27b0f66a56)

Here are three screenshots showing all the components of my network.
[https://gyazo.com/f451dff7acc7e97de12445fd4d9116bf](https://gyazo.com/f451dff7acc7e97de12445fd4d9116bf)
[https://gyazo.com/6f16e77f8a7112db25bb779f0f71078d](https://gyazo.com/6f16e77f8a7112db25bb779f0f71078d)
[https://gyazo.com/6e06cbecf3c977f46c19429c5948b30c](https://gyazo.com/6e06cbecf3c977f46c19429c5948b30c)

Forge: 36.0.35
AE2: 8.2.0-beta.2
AE2 extras: 1.2

I am playing All the mods 6 in singleplayer. Let me know if you need any additional information

**Comment by Tfarcenim** [comment]
Hmm, what happens if you isolate to just these 2 mods and the forge version, I'm currently running with 8.2.0 and not able to reproduce the breaking.

Also, is there anything in the log?

**Comment by MilkMC** [comment]
I did some more testing and found that it only happens when a security terminal is on the network. Here is a gif demonstrating that. [https://gyazo.com/1631a544fac8f39d1c37acd42adda637](https://gyazo.com/1631a544fac8f39d1c37acd42adda637)

I also tested it separately with just those 2 mods and forge and got the same result. I went ahead and tested all the crafting storages added by your mod and all 4 have the issue.

Nothing appears in the log.

I will edit the title to reflect the security terminal requirement.
